recipes/pycoxmunk/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "pycoxmunk" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycoxmunk-{{ version }}.tar.gz
+  sha256: 88eff51543699e208d3c2d9e7937b864ac820cf33ddc622e9c071b93952b63c3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.21.2
+    - scipy >=1.7.1
+    - pandas >=1.3.4
+    - dask-core >=2021.10.0
+    - pyorbital >=1.7.1
+    - pyresample >=1.22.1
+    - satpy >=0.30
+    - click >=8.0.3
+    - numba >=0.55.0
+    - xarray >=0.20.2
+
+test:
+  imports:
+    - pycoxmunk
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/simonrp84/PyCoxMunk
+  summary: An algorithm to compute sea surface reflectance using the Cox-Munk approach.
+  license: GPL-3.0
```

To fix up the metadata:
```suggestion
  license: GPL-3.0-or-later
```